src/agor/tools/dev_tooling.py

```
@@ -233,114 +233,151 @@ def display_git_workflow_status() -> str:
 # ===========================
 
 
-def list_memory_branches(repo_path: Optional[Path] = None) -> List[str]:
+# Import memory branch utilities from memory_manager to avoid code duplication
+from agor.tools.memory_manager import read_from_memory_branch, list_memory_branches
```

Ruff (0.11.9)

`agor.tools.memory_manager.read_from_memory_branch` imported but unused

Remove unused import

(F401)